pkg/admission/validate_noobaa.go

```diff
@@ -0,0 +1,45 @@
+package admission
+
+import (
+	"github.com/sirupsen/logrus"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewNoobaaValidator initializes a NoobaaAccountValidator to be used for loading and validating a noobaaaccount
+func NewNoobaaValidator(arRequest admissionv1.AdmissionReview) *ResourceValidator {
+	nav := &ResourceValidator{
```

Comment:
In noobaaaccount validator the nav stand for noobaa account validator.
We can change it here to be nv (and also in all the other places.)
```suggestion
	nv := &ResourceValidator{
```

Reply:
Fixed ✅